gke-disk-image-builder/cli/main.go

```diff
@@ -61,6 +62,10 @@ func main() {
 		log.Panicf("invalid argument, timeout: %v, err: %v", timeout, err)
 	}
 
+	if len(*jobName) > 50 {
+		log.Panicf("invalid argument, job-name: %v cannot be longer than 50 characters, got: %v", *jobName, len(*jobName))
```

Comment:
just curious, is this limit from daisy?

Reply:
This is the limit introduced by GCP resource naming convention. We are using "${jobName}-bootable-disk" as one of the disk names and that one can't go over 63 characters, so jobName should be < 50 (counting again I think I should check for if `len(*jobName) >= 50` -- I'll update)